libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -20,42 +20,46 @@
 LOG = get_logger('analyzer')
 
 
+def parse_checkers(clangsa_output):
+    """
+    Parse clang static analyzer checkers list output.
+    Return a list of (checker name, description) tuples.
+    """
+
+    # Checker name and description in one line.
+    pattern = re.compile(
+        r'^\s\s(?P<checker_name>\S*)\s*(?P<description>.*)')
```

Comment:
What if it contains more than two spaces at the beginning? Wouldn't it be better to write `^\s*`?

Reply:
I think we can change it (the output did not change in the last years), but I would do it in a separate pull request.